source/dialogs/window.c

```diff
@@ -332,20 +333,26 @@ static client* window_client ( ModeModePrivateData *pd, xcb_window_t win )
         xcb_ewmh_get_atoms_reply_wipe ( &states );
     }
 
-    c->title = window_get_text_prop ( c->window, xcb->ewmh._NET_WM_NAME );
-    if ( c->title == NULL ) {
-        c->title = window_get_text_prop ( c->window, XCB_ATOM_WM_NAME );
+    char *tmp_title = window_get_text_prop ( c->window, xcb->ewmh._NET_WM_NAME );
+    if ( tmp_title == NULL ) {
+        tmp_title = window_get_text_prop ( c->window, XCB_ATOM_WM_NAME );
     }
+    c->title      = g_markup_escape_text ( tmp_title, -1 );
```

Comment:
Hmm.. is it possible to tmp_title to still be NULL here? Should it maybe be `tmp_title ? tmp_title : ""`?